contentcuration/contentcuration/frontend/settings/pages/Account/StudioCopyToken.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+
+  <div class="token-input-wrapper">
+    <KCircularLoader
+      v-if="loading"
+      size="38"
+      class="loader"
+    />
+    <template v-else>
+      <KTextbox
+        ref="tokenInput"
```

Comment:
I haven't noticed this `ref` being used from anywhere. Can you check and delete?